### DISCUSSION TITLE
Removing a foreign key - best practices

### DISCUSSION THREAD

**Discussion by donnyxray** [rqlite/rqlite#1953] [General]
What is the best way to remove a foreign key constraint from an rqlite table?

Doing so with sqlite requires [some trickery](https://www.sqlite.org/lang_altertable.html#otheralter). This procedure suggests to start a transaction, which I think may not work as expected when using rqlite?

**Comment by otoolep** [comment]
Just perform steps 3 to 10 inclusive in a bulk request, with the transaction flag to true. That should do it.

- https://rqlite.io/docs/api/bulk-api/
- https://rqlite.io/docs/api/bulk-api/#transaction-support
- https://rqlite.io/docs/api/api/#transactions

Make sense?
